runtime/doc/news.txt

```diff
@@ -60,6 +60,9 @@ The following new APIs or features were added.
 
   See https://github.com/neovim/neovim/pull/14537.
 
+• |vim.secure.trust()|, |:trust| allows the user to manage files in trust
+  database.
+
```

Comment:
Move this to the top of the list, so far we've been adding features to news.txt in reverse-chronological order (newest features on top)

Reply:
Done in a87789fd1a4710209d3558f6314ea40713d43ddf